modules/ROOT/pages/getting-started-tutorial.adoc

```diff
@@ -0,0 +1,307 @@
+            "name": "autologin-core.conf"
+          }
+        ],
+        "name": "serial-getty@ttyS0.service"
+      }
+    ]
+  }
+}
+----
+
+Luckily fcct outputs valid Ignition configs.
+However, if you are tweaking the config after fcct, or otherwise providing hand edited Ignition configs you'll want to always use ignition-validate to perform some verification on the config.
+We'll do that now to illustrate how:
+
+[source,bash]
+----
+[host]$ ignition-validate --version
+Ignition 2.4.1
+[host]$ ignition-validate ./simple.ign && echo 'success!'
+----
+
+== Booting Fedora CoreOS: Simple Provisioning Scenario
+
+Given that we've now created an Ignition config. Let's try to boot a VM with an image and that config.
+For this lab we'll use the qemu image but you should be able to use that Ignition config with any of the artifacts that are published for a Fedora CoreOS release.
+
+In this case we'll use libvirt/qemu/kvm to boot directly into Fedora CoreOS from the qemu image.
+
+[source,bash]
+----
+[host]$ chcon --verbose unconfined_u:object_r:svirt_home_t:s0 simple.ign
+[host]$ virt-install --name=fcos --vcpus=2 --ram=2048 --import \
+            --network=bridge=virbr0 --graphics=none \
+            --qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${PWD}/simple.ign" \
+            --disk=size=20,backing_store=${PWD}/fedora-coreos[stream_version].qcow2
+----
+
+*Note:* You would have to replace [stream-version] with the archive name.
+
+*Note:* We used chcon here to set appropriate SELinux file contexts on the Ignition config file so that the resulting process would be able to access the file.
+
+The `virt-install` command will start an instance named `fcos` from the `fedora-coreos-[stream_version]` image using the `simple.ign` Ignition config.
+It will auto-attach the serial console of the machine so you'll be able to see the image bootup messages.
+Also of note is that it uses the `backing_store` option to `virt-install --disk` so it won't write to the download image, but rather a new disk image that can be easily thrown away.
+
+Once the machine is booted up you should see a few promps and then you should be automatically logged in and presented with a bash shell:
+
+[source,bash]
+----
+[  OK  ] Started RPM-OSTree System Management Daemon.
+
+Fedora CoreOS preview [stream_version]
+Kernel 5.4.7-200.fc31.x86_64 on an x86_64 (ttyS0)
+
+eth0: 192.168.122.20 fe80::5054:ff:feb2:e7f9
+localhost login: core (automatic login)
+
+[core@localhost ~]$
+----
+
+*Note:* IP address and Kernel version can differ.
+
+Magic? Yes! The systemd dropin was created and we were automatically logged in to the terminal.
+You can see the full configuration for serial-getty@ttyS0.service by using systemctl cat serial-getty@ttyS0.service.
```

Comment:
systemctl cat serial-getty@ttyS0.service -> we need a code block here (I don't know how they are made in asciidoc.

Reply:
here: https://github.com/coreos/fedora-coreos-docs/pull/134/commits/ab28d5d0efacf9c71cafcac66129c5a7af4ea69c